__tests__/app.test.js

```diff
@@ -70,4 +70,51 @@ describe("/api/articles/:article_id", () => {
             expect(body.msg).toBe("Bad Request Invalid Article ID");
           });
     });
+});
+
+
+describe("POST /api/articles/:article_id/comment", () => {
+    test("POST: 201 responds with posted comment", () => {
+    const newComment = {
+        username: "butter_bridge", body: "test comment"};
+    return request(app)
+        .post(`/api/articles/1/comments`)
+        .send(newComment)
+        .expect(201)
+        .then(({ body }) => {
+        const { postedComment } = body;
+        expect(postedComment).toMatchObject({
+            comment_id: expect.any(Number),
+            article_id: expect.any(Number),
+            author: "butter_bridge",
+            body: "test comment",
+            created_at: expect.any(String),
+            votes: expect.any(Number)
+        })
+    })
+    })
+    test("POST: 400 responds with an error message of 'Bad Request' if passed invalid user", () => {
+        const newComment = { 
+            username: "billybob",
+        }
+        return request(app)
+        .post("/api/articles/1/comments")
+        .send(newComment)
+          .expect(400)
+          .then(({ body }) => {
+            expect(body.msg).toBe("Bad Request Body");
+          });
+    });
+    test("POST: 400 responds with an error message of 'Bad Request' if passed no comment", () => {
+        const newComment = { 
+            username: "butter_bridge",
+        }
+        return request(app)
+        .post("/api/articles/1/comments")
+        .send(newComment)
+          .expect(400)
+          .then(({ body }) => {
+            expect(body.msg).toBe("Bad Request Body");
+          });
+    })
```

Comment:
There are a few more tests needed here - 
Status 201 - ignores unnecessary properties (for example, `username : "test", body : "test", extraProp : "test"`
Status 404 - invalid username
Status 404 - id doesn't exist 
Status 400 - invalid id